distrho/src/ladspa/ladspa.h

```diff
@@ -30,6 +30,14 @@
 extern "C" {
 #endif
 
+#ifndef LADSPA_PLUGIN_EXPORT
```

Comment:
modifying the ladspa header is not needed, and thus should be avoided.
we manually set this symbol for export ourselves.

Reply:
The definition enters in conflict with the declaration, because one of these has dllexport and the other hasn't.

Reply:
huh that is new to me. with gcc we can redeclare functions to change their visibility status.
so we need the same for DSSI as well, right?

Reply:
Yes it applies to DSSI, so I did the relevant edit (if DSSI ever matters on windows platforms)